src/config/Config.ts

```diff
@@ -41,7 +37,26 @@ export class Config {
     return Config.internalInstance.initPromise;
   }
 
-  public config: ResolvedConfigOptions;
+  // Convenience accessors
+  public static defaultHomeserverUrl(): string | undefined {
+    const defaultServerConfig = Config.instance.config.default_server_config;
+    if (!defaultServerConfig) {
+      return undefined;
+    }
+
+    return defaultServerConfig["m.homeserver"]?.base_url;
```

Comment:
The types say that `default_server_config` and `m.homeserver` are always defined - however `Config.instance.config` is the thing that actually might not be